bnd-maven-plugin/.gitignore

```diff
@@ -1 +1,4 @@
+/.classpath
+/.project
+/.settings/
```

Comment:
Correct that is my understanding as well, let all eclipse metadata come
downstream of pom.xml.  however, some Eclipse metadata can't come
downstream of pom.xml like java code style.  For now I'll pull my commit of
that metadata until you guys decide.

On Thu, Apr 9, 2015 at 5:01 PM, Neil Bartlett <notifications@github.com>
wrote:

> In bnd-maven-plugin/.gitignore
> <https://github.com/bndtools/bnd/pull/910#discussion_r28045646>:
>
> > @@ -1 +1,4 @@
> > +/.classpath
> > +/.project
> > +/.settings/
>
> As I understand, the convention in maven projects is to NOT check in those
> files. This does differ from other bnd projects.
> … <#14c9d6a0e082818b_>
> -- Neil Bartlett Sent from a phone
> On Thursday, 9 April 2015 at 09:34, Gregory Amerson wrote: In
> bnd-maven-plugin/.gitignore (
> https://github.com/bndtools/bnd/pull/910#discussion_r28043828): > @@ -1
> +1,4 @@ > +/.classpath > +/.project > +/.settings/ So in order to work on
> the "bnd-maven-plugin" project I imported into my eclipse using m2e plugin,
> which fires off the project configurators and generates those files. So
> some developers will ignore these files since you can get them back
> whenever you "import" the project using m2e. some people want these files
> checked in. Both ways work I guess, what is bndtools pref and I can follow
> that way. — Reply to this email directly or view it on GitHub (
> https://github.com/bndtools/bnd/pull/910/files#r28043828).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bndtools/bnd/pull/910/files#r28045646>.
>



-- 
Greg Amerson
Liferay Developer Tools
Liferay, Inc. www.liferay.com